### ISSUE TITLE
Accept alphanumeric entity_id in show_controllers option.

### ISSUE THREAD

**Issue by gufonero** [rgc99/irrigation-unlimited-card#17]
**Is your feature request related to a problem? Please describe.**
The show_controllers option, which specifies which controllers should be shown in the card, takes a CSV list of controllers identified by number; entity_id's are not valid elements in that list.

**Describe the solution you'd like**
I propose that alphanumeric entity_id's be also valid values to specify a controller in the CSV list.

**Describe alternatives you've considered**
Using the numeric id works, but it is error-prone especially if the controllers' definitions or their order are changed.

**Additional context**
As a side note, this option is listed as 'controllers' in section 3 (Options) of the README file; I think it should be 'show_controllers'.

**Comment by gufonero** [comment]
Thank you for the fix, Robert; I was not expecting such a quick reaction. Unfortunately it does not work, at least not for me. 

Just to be clear: I updated my local git repo from GitHub to the latest commit (the one following this fix); I created the bundle and installed the js card from the dist directory into my HA setup. Then I changed the show_controllers in the IU card config to the entity_id of the desired controller as specified in the controller_id entry of the controller's definition. I tried both with the controller_id and with the controller_id followed by a comma (to force it to a csv list, as needed when the position number is used). Only the top title with the four top icons is shown with this setup, wheras it works fine with the position number.

In the browser console log I saw no errors that I could relate to the IU card.

I wonder if the option rename_entities that I have set to true might be related to this behaviour. I think I have read something about it not being fully compatible with the card, but I couldn't find it again to check.
